docs/core/eventdata/measurements.md

```diff
@@ -0,0 +1,12 @@
+# Measurements
+
+Measurements exist as *uncalibrated* and *calibrated* types. While
+*uncalibrated* measurements are the direct output of the data formation, during
+the track fit, when trajectory information (or information about the trajectory
+hypothesis) is available, certain calibration steps can help to improve the
+track fit quality.
```

Comment:
The second sentence is not really clear to me.

Reply:
these are changes I absorbed from paul and it is just moving the text from `docs/core/eventdata.md` to here

but I can try to improve it